examples/account-management.ipynb

```diff
@@ -0,0 +1,264 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_runtime import IBMRuntimeService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Examples\n",
+    "\n",
+    "## Account Management\n",
+    "### Load account"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# load default account (heuristic to be determined)\n",
+    "account1 = IBMRuntimeService.load_default_account()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# load named account from default JSON configuration file\n",
+    "account2 = IBMRuntimeService.load_account(name=\"my-account\")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# load account configured via environment variables\n",
+    "account3 = IBMRuntimeService.load_account_from_environment()"
```

Comment:
This should be just
```
service = IBMRuntimeService()
```
If env variables are set they should take preference over credentials stored in json file.